homeassistant/components/keba/sensor.py

```diff
@@ -0,0 +1,113 @@
+"""Support for KEBA charging station sensors."""
+import logging
+
+from homeassistant.const import ENERGY_KILO_WATT_HOUR
+from homeassistant.helpers.entity import Entity
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_platform(hass, config,
+                               async_add_entities, discovery_info=None):
+    """Set up the KEBA charging station platform."""
+    _LOGGER.debug("Initializing KEBA charging station sensors")
+    keba = hass.data[DOMAIN]
+
+    sensors = [
+        KebaSensor(keba, 'Curr user', 'Max current', 'mdi:flash',
+                   'A'),
+        KebaSensor(keba, 'Setenergy', 'Energy target', 'mdi:gauge',
+                   ENERGY_KILO_WATT_HOUR),
+        KebaSensor(keba, 'P', 'Charging power', 'mdi:flash',
+                   'kW'),
```

Comment:
This sensor can use the device class `'power'`.
https://developers.home-assistant.io/docs/en/entity_sensor.html#available-device-classes

Reply:
I added an optional argument for the device class in generation of the sensors